federatedscope/llm/model/model_builder.py

```diff
@@ -90,4 +42,14 @@ def get_llm(config):
         input_embeddings[-num_new_tokens:] = input_embeddings_avg
         output_embeddings[-num_new_tokens:] = output_embeddings_avg
 
+    use_adapter = config.llm.adapter.use
+    if use_adapter:
+        adapter_package = config.llm.adapter.args[0]['adapter_package']
+        adapter_method = config.llm.adapter.args[0]['adapter_method']
+    else:
+        adapter_package = None
+        adapter_method = None
+
+    model = AdapterModel(model, use_adapter, adapter_package, adapter_method)
```

Comment:
you can also pass the arg `config.llm.adapter.args[0]` for user specified paras as mentioned above

Reply:
Done.